src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,84 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int RESIZE_FACTOR = 2;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] myData;
+    private int size;
+
+    public MyHashMap() {
+        myData = new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= myData.length * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
+        Node<K, V> currentNode = myData[getIndex(key)];
```

Comment:
How about creating a local variable for `getIndex(key)` method so that we improve code readibility?